mungegithub/mungers/sig-mention-handler.go

```diff
@@ -102,11 +111,11 @@ func (s *SigMentionHandler) Munge(obj *github.MungeObject) {
 	hasNeedsSigLabel := s.HasNeedsSigLabel(obj)
 
 	if hasSigLabel && hasNeedsSigLabel {
-		if err := obj.RemoveLabel("needs-sig"); err != nil {
+		if err := obj.RemoveLabel(needsSigLabel); err != nil {
 			glog.Errorf("failed to remove needs-sig label for issue #%v", *obj.Issue.Number)
```

Comment:
Will `#%d` a better option here?

Reply:
@cblecker We should use `%d` as `*obj.Issue.Number` is an integer. Is there a specific reason for using `%v` here?